core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java

```diff
@@ -417,13 +417,13 @@ private void setMailProps() {
          * set default value
          */
         if(DATAINFO.getProperty("mailSmtpStarttls.required") == null) {
-        	DATAINFO.setProperty("mail.smtp.starttls.required", "true");
+        	DATAINFO.setProperty("mail.smtp.starttls.required", "");
         }else {
         	DATAINFO.setProperty("mail.smtp.starttls.required", DATAINFO.getProperty("mailSmtpStarttls.required"));
         }
         
         if(DATAINFO.getProperty("mailSmtpSslProtocols") == null) {
-        	DATAINFO.setProperty("mail.smtp.ssl.protocols", "TLSv1.2");
+        	DATAINFO.setProperty("mail.smtp.ssl.protocols", "");
```

Comment:
Setting them to blank might have an undesirable effect. I would just not set the properties at all. E.g. I would change the if-else block here to:
`if(DATAINFO.getProperty("mailSmtpSslProtocols") != null) {
DATAINFO.setProperty("mail.smtp.ssl.protocols", DATAINFO.getProperty("mailSmtpSslProtocols"));
}
`

Reply:
It looks like there are two ways of setting these properties. One via spring using applicationContext-email.xml and other in EmailEngine.props. The ones that are set here need to be read in EmailEngine class and added to EmailEngine.props. 
For emails sent using spring, you need to set the defaults within applicationContext-email.xml
E.g. `<prop key="mail.smtp.ssl.protocols">s[mail.smtp.ssl.protocols: TLSv1.1]</prop>`
Reference: https://stackoverflow.com/a/2534949

Reply:
code change like below: we should set these properties only if the customer explicitly sets them. Otherwise we will use default setting of these 2 properties in applicationContext-core-email.xml.